runtime/Cargo.toml

```diff
@@ -15,6 +15,7 @@ fnv = "1.0.6"
 hashbrown = "0.1.8"
 log = "0.4.2"
 rand = "0.6.5"
+rayon = "1.0.3"
```

Comment:
Are you using this?

Reply:
Oh dang, I was earlier. I'll get rid of it asap. Thanks!